src/rules/no_private_function_fat_arrows.coffee

```diff
@@ -0,0 +1,44 @@
+
+module.exports = class NoPrivateFunctionFatArrows
+
+    rule:
+        name: 'no_private_function_fat_arrows'
+        level: 'warn'
+        message: 'Used the fat arrow for a private function'
+        description: """
+            Warns when you use the fat arrow for an object method. This is
```

Comment:
I think this needs a better explanation, but I'm not exactly sure what it should say. "object method" would seem to indicate that it's `Foo.bar()` but this isn't really for methods on objects. It's for functions that are closured into the class definition scope.